config/config/sc-config.yaml

```diff
@@ -11,7 +11,6 @@ global:
 objectStorage:
   buckets:
     harbor: ${CK8S_ENVIRONMENT_NAME}-harbor
-    influxDB: ${CK8S_ENVIRONMENT_NAME}-influxdb
```

Comment:
There is some updated alert rules that uses this bucket, I'll handle them in my PR since I've already modifed them so you know.